make/toolchain-android.cmake

```diff
@@ -47,9 +47,7 @@ set(CMAKE_SYSTEM_VERSION 1)
 if (NOT DEFINED TARGET_ABI)
   set(TARGET_ABI "arm-linux-androideabi")
 endif ()
-if (TARGET_ABI MATCHES "^arm")
-  set(CMAKE_SYSTEM_PROCESSOR arm)
-endif ()
+set(CMAKE_SYSTEM_PROCESSOR arm)
```

Comment:
I know, I just wanted to try if it fixes the issue on the CI. I'll investigate a cross-build using the Android gcc shipped with Ubuntu.